buildenv/jenkins/getDependency

```diff
@@ -22,17 +22,17 @@ def testBuild() {
 				sh 'tar -xzf OpenJDK8U-jdk_x64_linux_hotspot*.gz -C ${WORKSPACE}/j2sdk-image --strip-components 1'
 				sh '${WORKSPACE}/j2sdk-image/jre/bin/java -version'
 				sh 'git clone https://github.com/AdoptOpenJDK/openjdk-systemtest'
-				sh 'git clone https://github.com/AdoptOpenJDK/stf'
+				sh 'git clone https://github.com/Mesbah-Alam/stf -b consolidate-prereq-version-numbers'
```

Comment:
Please remove the personal repo and branch.

Reply:
Ah. That one was missed! Thanks for the catch! Updated.